twemproxy_exporter.gemspec

```diff
@@ -1,7 +1,5 @@
 # coding: utf-8
-lib = File.expand_path('../lib', __FILE__)
-$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
-require 'twemproxy_exporter/version'
+require_relative 'lib/twemproxy_exporter/version'
```

Comment:
I'd rather not use `require_relative` here. When rubygems loads the gem, the `lib` directory should get added to the load path. This code was generated by bundler when the gem was built using `bundle gem twemproxy_exporter`.

It probably doesn't matter too much for this gem specifically, but in general I think it's cleaner and more idiomatic to add the lib dir to the load path and then use regular `require` statements.